examples/example-marketing/package.json

```diff
@@ -25,7 +26,6 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-hook-form": "^7.8.6",
```

Comment:
@alex-ahumada Minor fix here. We have an extra trailing comma.

Reply:
Done!